charts/rd-user-profile-api/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "0.1"
 description: Profile service for all users
 name: rd-user-profile-api
-version: 0.0.7
+version: 0.0.8
```

Comment:
we are not doing charts related updated, hence please keep same version.

Reply:
will I leave them the same for each one? this is getting a bit ridiculous, I dunno if i should change or not

Reply:
for each branch on each repo